lib/reek/sexp/sexp_formatter.rb

```diff
@@ -8,14 +8,15 @@ module Sexp
     #
     # Formats snippets of syntax tree back into Ruby source code.
     #
+    # :reek:DuplicateMethodCall { max_calls: 2 } is ok for lines.first
     class SexpFormatter
       def self.format(sexp)
         return sexp.to_s unless sexp.is_a? AST::Node
         lines = Unparser.unparse(sexp).split "\n"
-        if lines.length > 1
-          "#{lines.first} ... #{lines.last}"
-        else
-          lines.first
+        case lines.length
+        when 1 then lines.first
+        when 2 then lines.join('; ')
+        else [lines.first, lines.last].join(' … ')
```

Comment:
I tried `lines.values_at(0, -1).join(' … ')` to avoid the duplicate method call, but I think it reads much worse. That's why I ignored the duplicate method call.